utility/namespace.py

```diff
@@ -0,0 +1,29 @@
+""" Create a namespace """
+import os
+from utility import templating
+from ocs.utils import create_oc_resource
+
+
+def create_namespace(name):
+    """
+    Create a namespace
+
+    When a new namespace is created, a directory in /tmp with the namespace's
+    name will be created containing the namespace.yaml file that created this
+    namespace.
+
+    Raises AlreadyExists exception if the namespace already exists.
+
+    Args:
+        name (str): Namespace to be created
+    """
+    env_data = dict()
+    env_data['cluster_namespace'] = name
+    local_dir = f"/tmp/{name}"
```

Comment:
I didn't understand the purpose of storing yaml locally, if we need any spec info related to the namespace we can extract it from "oc get -o yaml" command. May be I am missing something.